engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/validation/ZeebeRuntimeValidators.java

```diff
@@ -151,6 +152,13 @@ public static Collection<ModelElementValidator<?>> getValidators(
             .hasValidExpression(Signal::getName, ExpressionVerification::isOptional)
             .build(expressionLanguage),
         // Checks signal name expressions of start event signals
-        new ProcessSignalStartEventSignalNameValidator(expressionLanguage));
+        new ProcessSignalStartEventSignalNameValidator(expressionLanguage),
+        // ----------------------------------------
+        ZeebeExpressionValidator.verifyThat(ZeebeError.class)
+            .hasValidExpression(
+                ZeebeError::getErrorCodeVariable, ExpressionVerification::isOptional)
+            .hasValidExpression(
+                ZeebeError::getErrorMessageVariable, ExpressionVerification::isOptional)
+            .build(expressionLanguage));
```

Comment:
❌ I don't think this is needed. The error code and message variables are just strings of static variable names. They are not expressions, so we don't have to verify that they are valid expressions.